code/modules/antagonists/abductor/equipment/abduction_gear.dm

```diff
@@ -362,7 +362,7 @@
 
 		to_chat(L, "<span class='italics'>You hear a voice in your head saying: </span><span class='abductor'>[message]</span>")
 		to_chat(user, "<span class='notice'>You send the message to your target.</span>")
-		log_talk(user,"[key_name(user)] sent an abductor mind message to [key_name(L)]: '[message]'", LOGSAY)
+		log_directed_talk(user, L, message, "abductor whisper")
```

Comment:
Missing message type

Reply:
The `message_type` is already checked in `log_message` (which all other logging functions call), where it throws a warning if it's not one of the `LOG_*` constants.

Reply:
I see, would be nice it stack_traced, it might be bit annoying to find if it's mixed argument order.

Reply:
I thought warnings already printed a stack trace, but I was wrong. Fixing.